homeassistant/components/linky/__init__.py

```diff
@@ -1 +1,43 @@
 """The linky component."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EVENT_HOMEASSISTANT_START
+from homeassistant.core import callback
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+_LOGGER.error("LINKY_INIT")
+
+
```

Comment:
Add a config schema here. The config section will move under the linky key in configuration.yaml.